src/components/about/AboutLayout.tsx

```
@@ -0,0 +1,483 @@
+						<div className="absolute flex h-12 w-12 items-center justify-center rounded-md bg-primary/45 text-foreground">
+							{p.Icon}
+						</div>
+						<p className="ml-16 text-xl font-cabin text-left font-extrabold leading-6 text-foreground">
+							{p.Title}
+						</p>
+					</dt>
+
+					<dd className="mt-3 ml-16 text-foreground text-monster">
+						<p
+							className="text-xs/4 md:text-sm lg:text-md font-normal font-inter opacity-80 text-justify"
+							dangerouslySetInnerHTML={{ __html: p.Description }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)